pkg/configurator/client.go

```diff
@@ -223,6 +226,9 @@ type osmConfig struct {
 	OutboundIPRangeExclusionList string `yaml:"outbound_ip_range_exclusion_list"`
 
 	EnablePrivilegedInitContainer bool `yaml:"enable_privileged_init_container"`
+
+	// EnableTicker is a flag to configure ticker for regular proxy broadcast updates
+	EnableTicker string `yaml:"enable_ticker"`
```

Comment:
I propose we rename this. (See other comment on `enable_ticker`)

Reply:
Should we make the type `time.Duration`?

Reply:
We'll do similar to ServiceCertValidityDuration, where the duration is parsed on the api side.